xdsl/dialects/func.py

```diff
@@ -322,6 +322,36 @@ def get(
             attributes={"callee": callee},
         )
 
+    def print(self, printer: Printer):
+        printer.print_string(" ")
+        printer.print_attribute(self.callee)
+        printer.print_string("(")
+        printer.print_list(self.arguments, lambda val: printer.print_ssa_value(val))
+        printer.print_string(")")
+        printer.print_op_attributes(self.attributes, reserved_attr_names=("callee",))
+        printer.print_string(" : ")
+        printer.print_operation_type(self)
+
+    @classmethod
+    def parse(cls, parser: Parser) -> Call:
+        callee = parser.parse_symbol_name()
+        unresolved_arguments = parser.parse_comma_separated_list(
+            Parser.Delimiter.PAREN, lambda: parser.parse_unresolved_operand()
```

Comment:
```suggestion
            Parser.Delimiter.PAREN, parser.parse_unresolved_operand
```
Actually, if I recall, wasn't there an operation that does exactly this?

Reply:
indeed